micrometer-registry-new-relic-telemetry/src/main/resources/META-INF/native-image/io.micronaut.configuration.metrics/micrometer-new-relic/native-image.properties

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
I think these files need to be placed in `META-INF/native-image/{groupId}/{artifactId}/native-image.properties`, otherwise they will produce a warning.

@graemerocher @ilopmar is this correct?

Reply:
Yes, what Álvaro said.

Also, we already have a `micrometer-registry-new-relic` module with package `io.micronaut.configuration.metrics.micrometer.newrelic` so we can't use the same package in this new module because we will have split packages.

The package for this module could be `io.micronaut.configuration.metrics.micrometer.newrelictelemetry` or something similar

Reply:
Okay. I took a shot at this one. Changed the package name as suggested and differentiated the native-image.properties path using same convention as other the other modules and referencing [GraalVM Build Configuration](https://www.graalvm.org/reference-manual/native-image/BuildConfiguration/.)

Reply:
I think the file path is still incorrect. You have put the file under:

```
.../io.micronaut.configuration.metrics/micrometer-new-relic-telemetry/native-image.properties
```

And I believe it should be:

```
.../io.micronaut.micrometer/micronaut-micrometer-registry-new-relic-telemetry/native-image.properties
```

Reply:
I'm trying to follow the convention used in the other registry projects in the repo. They all omit 'registry' from the artifact name. 

I as I understand it, it is convention, not a requirement/syntax, to use the actual artifact name here.

If I change it for this registry I think the change should be applied across the board for the sake of consistency. 

Do you agree? If not, I'll go ahead and push the change to this registry only, per your comment.

See below:
```
.../io.micronaut.configuration.metrics/micrometer-statsd/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-dynatrace/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-azure-monitor/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-new-relic-telemetry/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-appoptics/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-influx/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-graphite/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-elastic/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-humio/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-cloudwatch/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-ganglia/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-signalfx/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-stackdriver/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-wavefront/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-atlas/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-prometheus/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-jmx/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-kairos/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-datadog/native-image.properties
.../io.micronaut.configuration.metrics/micrometer-new-relic/native-image.properties
```

Reply:
You are right that is a convention only, but produces a warning that might confuse users, so it's best to avoid it if we can.

Also, I think this warning was introduced in recent versions of GraalVM, that's why the other properties may not be following the convention.

Thinking about it, @ilopmar wouldn't it be better to have an `AutomaticFeature` instead of those properties?

Reply:
I think it's easier to keep a `native-image.properties` with the initialization for every module inside the modules that a common file with a lot of `if`s to check whether every class is present. 
My rule of thumb for this is: "If I can do it with config files, I go this way. If I need some condition, then I go with an `@AutomaticFeature` (like we did in `micronaut-sql`)"

Reply:
Fair enough. @mparlee if you could proceed with renaming the paths of the other files, it would be awesome.